ezomero/_gets.py

```diff
@@ -104,6 +109,13 @@ def get_image(conn, image_id, no_pixels=False, start_coords=None,
         if type(pyramid_level) is not int:
             raise TypeError('pyramid_level must be an int')
 
+    if dim_order is not None:
+        if type(dim_order) is not str:
+            raise TypeError('dim_order must be a str')
+        if set(dim_order.lower()) != set('xyzct'):
+            raise ValueError('dim_order must contain letters \
```

Comment:
this error message has a newline in it, so it print's really weird when throwing an exception. You don't actually need the `\` there...you should just have two closed strings in the parenthesis like:

```
raise ValueError('dim_order must contain letters'
                 'xyzct exactly once')
```

Reply:
except I am missing a space!